Sources/Controller/Abstract/ModesBaseTableViewController.swift

```diff
@@ -37,21 +37,21 @@ class ModesBaseTableViewController: UITableViewController {
         title = Strings.Titles.Modes
     }
     
-    override func viewWillAppear(animated: Bool) {
-        self.clearsSelectionOnViewWillAppear = self.splitViewController!.collapsed
+    override func viewWillAppear(_ animated: Bool) {
+        self.clearsSelectionOnViewWillAppear = self.splitViewController!.isCollapsed
```

Comment:
Re. `self.splitViewController!` 

Is it still annotated by Cocoa such that force unwrap is necessary?